x-pack/plugins/security_solution/public/overview/components/overview_host_stats/__snapshots__/index.test.tsx.snap

```diff
@@ -28,7 +28,7 @@ exports[`Overview Host Stat Data rendering it renders the default OverviewHostSt
         <EuiFlexItem
           grow={false}
         >
-          <Memo(StatValueComponent)
+          <UNDEFINED
```

Comment:
Does this change seem right?

Reply:
Good catch, I pinned the react-is version in x-pack and that resolved this.